paddlenlp/transformers/llama/modeling.py

```diff
@@ -605,8 +605,9 @@ def apply_rotary_pos_emb(q, k, cos, sin, position_ids):
 
 
 class LlamaMLP(nn.Layer):
-    def __init__(self, config):
+    def __init__(self, config, skip_recompute_ops={}):
```

Comment:
```suggestion
    def __init__(self, config, skip_recompute_ops=None):
        if skip_recompute_ops is None:
            skip_recompute_ops = {}
```
`skip_recompute_ops={}` 这种写法很容易有问题。

Reply:
https://blog.arisa.moe/blog/2022/220902-python-mutable-defaults/

Reply:
DONE